warp10/src/main/java/io/warp10/plugins/influxdb/InfluxDBWarp10Plugin.java

```diff
@@ -94,18 +103,21 @@ public void init(Properties properties) {
     this.port = Integer.parseInt(properties.getProperty(CONF_INFLUXDB_PORT, "8086"));
     this.host = properties.getProperty(CONF_INFLUXDB_HOST, "127.0.0.1");
     this.token = properties.getProperty(CONF_INFLUXDB_DEFAULT_TOKEN);
-    
+    this.measurementlabel = properties.getProperty(CONF_INFLUXDB_MEASUREMENT_LABEL);
+    this.threshold = Long.parseLong(properties.getProperty(CONF_INFLUXDB_FLUSH_THRESHOLD, DEFAULT_FLUSH_THRESHOLD));
+    this.cachesize = Integer.parseInt(properties.getProperty(CONF_INFLUXDB_CACHE_SIZE, DEFAULT_CACHE_SIZE));
+
     try {
       this.url = new URL(properties.getProperty(CONF_INFLUXDB_WARP10_ENDPOINT));
```

Comment:
Improve error message when connection to warp10 endpoint fails.

Reply:
At this step it's the URL syntax that's checked, no connection is attempted. What improvement did you have in mind?